src/modules/olMap/handler/mfp/OlMfpHandler.js

```diff
@@ -43,7 +42,11 @@ export class OlMfpHandler extends OlLayerHandler {
 		this._map = null;
 		this._registeredObservers = [];
 		this._pageSize = null;
-		this._lastCenter = [-1, -1];
+		this._visibleViewport = null;
+
+		const { srid: mfpSrid } = this._mfpService.getCapabilities();
```

Comment:
At this moment `this._mfpService.getCapabilities();` returns NULL because it is not yet initialized

Reply:
got it